<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-10T10:03:20Z" name="Default suite" finished-at="2024-07-10T10:03:54Z" duration-ms="33396">
    <groups>
      <group name="Error validation">
        <method signature="ValidationError.landingPageValidation()[pri:0, instance:ReneeDish.Tests.ValidationError@3fc9504b]" name="landingPageValidation" class="ReneeDish.Tests.ValidationError"/>
      </group> <!-- Error validation -->
    </groups>
    <test started-at="2024-07-10T10:03:20Z" name="Default test" finished-at="2024-07-10T10:03:54Z" duration-ms="33396">
      <class name="ReneeDish.Tests.ValidationError">
        <test-method is-config="true" signature="LaunchApplication()[pri:0, instance:ReneeDish.Tests.ValidationError@3fc9504b]" started-at="2024-07-10T10:03:20Z" name="LaunchApplication" finished-at="2024-07-10T10:03:40Z" duration-ms="19604" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method signature="cartpageValidation()[pri:0, instance:ReneeDish.Tests.ValidationError@3fc9504b]" started-at="2024-07-10T10:03:40Z" name="cartpageValidation" finished-at="2024-07-10T10:03:40Z" duration-ms="8" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "ReneeDishPractice.LandingPage.loginApplication(String, String)" because "this.landPage" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "ReneeDishPractice.LandingPage.loginApplication(String, String)" because "this.landPage" is null
at ReneeDish.Tests.ValidationError.cartpageValidation(ValidationError.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cartpageValidation -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:ReneeDish.Tests.ValidationError@3fc9504b]" started-at="2024-07-10T10:03:40Z" name="teardown" finished-at="2024-07-10T10:03:40Z" duration-ms="142" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="LaunchApplication()[pri:0, instance:ReneeDish.Tests.ValidationError@3fc9504b]" started-at="2024-07-10T10:03:40Z" name="LaunchApplication" finished-at="2024-07-10T10:03:53Z" duration-ms="13496" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method signature="landingPageValidation()[pri:0, instance:ReneeDish.Tests.ValidationError@3fc9504b]" started-at="2024-07-10T10:03:53Z" name="landingPageValidation" finished-at="2024-07-10T10:03:53Z" duration-ms="3" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "ReneeDishPractice.LandingPage.loginApplication(String, String)" because "this.landPage" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "ReneeDishPractice.LandingPage.loginApplication(String, String)" because "this.landPage" is null
at ReneeDish.Tests.ValidationError.landingPageValidation(ValidationError.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageValidation -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:ReneeDish.Tests.ValidationError@3fc9504b]" started-at="2024-07-10T10:03:53Z" name="teardown" finished-at="2024-07-10T10:03:54Z" duration-ms="125" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- ReneeDish.Tests.ValidationError -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
